drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/widget/analysis/cache/RuleInspector.java

```diff
@@ -56,6 +71,44 @@ public RuleInspector( final Rule rule,
 
             patternInspectorList.add( patternInspector );
         }
+
+        updateBRLActionInspectors( rule.getActions()
+                                       .where( Action.superType().is( ActionSuperType.BRL_ACTION ) )
+                                       .select().all() );
+        rule.getActions()
+            .where( Action.superType().is( ActionSuperType.BRL_ACTION ) )
+            .listen().all( new AllListener<Action>() {
```

Comment:
@Rikkola I find lamdas work OK (on the whole, although IIRC @porcelli had an issue with one use-case).. none of the Java 8 "stream" API is implemented in GWT2.8.0.beta1 so any use of that client-side is broken (but it's available in newer GWT2.8's)